package.json

```diff
@@ -6,17 +6,17 @@
   "license": "MIT",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha test/*.js test/block-management test/raw-transactions test/reducers test/transport test/utils",
-    "lint": "eslint src test/*.js test/block-management test/raw-transactions test/reducers test/transport test/utils",
+    "test": "mocha test",
+    "lint": "eslint src test",
```

Comment:
I'm curious about this change.  Was it the old way because mocha wasn't recursive but the version bump of mocha adds support for recursive test locating?

Reply:
No, both are recursive -- it's because we used to build a special browser testing bundle in the test directory, which wasn't intended to be run by mocha.  There's no reason to have that bundle just sitting in the directory though (should be generated as needed) so I removed it.